### DISCUSSION TITLE
Not a bug - Themes

### DISCUSSION THREAD

**Discussion by Readus-01** [mealie-recipes/mealie#2512] [Q&A]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.
- [ ] This issue can be replicated on the demo site (https://demo.mealie.io/)

### What is the issue you are experiencing?

have the theme and colour changes been removed from the latest version as the ability to do this, i can not find.
the app is working perfectly but i also cannot add recipes to categories 

### Steps to Reproduce

docker build using portainer

### Please provide relevent logs

none

### Mealie Version

1.0 beta 5

### Deployment

Docker (Synology)

### Additional Deployment Details

docker - portainer on synology

**Comment by hay-kot** [comment]
Theming has moved to by instance configuration for the sake of developer simplicity

See https://nightly.mealie.io/documentation/getting-started/installation/frontend-config/#themeing

Though, I'm not sure a lot of users are using it so it worked last time i checked, so please report any bugs you find!

**Reply by netsec007** [reply]
Link doesn't show top correctly

**Reply by netsec007** [reply]
404 not found

**Reply by boc-the-git** [reply]
There was a change to themes merged just yesterday that might make this link redundant.
You could check merged PRs to find it, or search the doco and you should find it. It's configured by environment variables.

**Reply by netsec007** [reply]
Yes, but the env variables are not working.  Its not picking them up.

**Reply by hay-kot** [reply]
If you're having an issue with the new theme changes, please fill out an issue with your deployment details using the issue forms. 

Otherwise we just have to guess what the problem is.

**Reply by boc-the-git** [reply]
And please make sure you update to the latest nightly; as I mentioned there was a change/fix released yesterday for the known theming issue.
https://github.com/mealie-recipes/mealie/pull/2688
